exp/urls.py

```diff
@@ -215,6 +216,11 @@
         StudyResponsesFrameDataCSV.as_view(),
         name="study-responses-download-frame-data-zip-csv",
     ),
+    path(
+        "studies/<int:pk>/responses/all/download_frame_zip_psychDS/",
+        StudyResponsesFrameDataPsychDS.as_view(),
+        name="study-responses-download-frame-data-zip-psychDS",
```

Comment:
Can we have the url be all lowercase?  `studies/<int:pk>/responses/all/download_frame_zip_psychds/` or something similar.